query-engine/connector-test-kit/src/test/scala/writes/topLevelMutations/UpsertMutationSpec.scala

```diff
@@ -511,7 +511,7 @@ class UpsertMutationSpec extends FlatSpec with Matchers with ApiSpecBase {
       """model TestModel {
           id        Int     @id
         |  optInt   Int?
-        |  optFloat Float?
+        |  optFloat Decimal?
```

Comment:
Heads up: different approach here. Happy to do the same as the other tests if it's preferred.

Reply:
that's okay.